users/service.go

```diff
@@ -215,12 +221,17 @@ func (svc usersService) User(ctx context.Context, token string) (User, error) {
 	}, nil
 }
 
-func (svc usersService) ListUsers(ctx context.Context, token string, groupID string, offset, limit uint64, um Metadata) (UserPage, error) {
-	_, err := svc.identify(ctx, token)
+func (svc usersService) Users(ctx context.Context, token string, offset, limit uint64, email string, m Metadata) (UserPage, error) {
+	aemail, err := svc.identify(ctx, token)
```

Comment:
Please add a comment that this is an admin email, because `aemail` looks confusing in the code.

Reply:
This is not really an admin email but simply an user email (or identity).